pywatts/core/pipeline.py

```diff
@@ -179,7 +179,10 @@ def _run(self, data: Union[pd.DataFrame, xr.Dataset], mode: ComputationMode):
         if isinstance(data, pd.DataFrame):
             data = data.to_xarray()
 
-        return self.transform(**{key: data[key] for key in data.data_vars})
+        if isinstance(data, xr.Dataset):
+            return self.transform(**{key: data[key] for key in data.data_vars})
+        else:
+            return self.transform(**data)
```

Comment:
Can you check that the dict contains dataarrays? And if data is neither a Dataset nor a Dict[DataArray] please raise an exception.

Reply:
Done!